### PR TITLE
better style and documentation

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,9 @@ Requests related to the funtionality or usability of the UI can be submitted as
     ```
 6. Code style:
     * Not in IntelliJ: use an eslint plugin.
-    * In IntelliJ: when you open the project, go to `Settings -> Editor -> Code Style -> Javascript`, click the gear next to `Scheme`, and import [js-style.xml](js-style.xml); then close Settings and right-click [.eslintrc.js](.eslintrc.js) and click `Apply Eslint Code Style Rules`.
+    * In IntelliJ:
```
Comment (ehanna4) [inline README.md:54]: I can't find a path that starts from 'Settings -> Editor _....', the most similar I can find is: 'File -> Other Settings -> Preferences for New Projects... -> Editor -> Code Style -> Javascript', not sure if this is something I am doing incorrectly though (tried looking for it with Help tab but to no avail) so would appreciate if someone else could verify if they can follow that path or not

Comment (cameron-ardell) [reply README.md:54]: I think it's now `Preferences -> Editor -> Code Style -> Javascript` (instead of `Settings` as the first word)

Comment (cameron-ardell) [reply README.md:54]: Also, this doesn't say what we're importing it as... there are a few choices when I look in IntelliJ: 
IntelliJ IDEA code style XML
Eclipse XML Profile
JSCS config file

I'm assuming it's either #1 or #3 and in either case, we need one more step about what we do with the import before it's finalized.

**Comment by cameron-ardell** [comment]
Ran this and followed all the steps and it's working a lot better, but is still reformatting indentation incorrectly sometimes on my computer.

**Review by ehanna4 — APPROVED** [review]
Changes are acceptable and more clear/correct

**Pull request merged**